test/WebJobs.Extensions.Tests/Extensions/Timers/Listener/TimerListenerTests.cs

```diff
@@ -358,49 +358,60 @@ public async Task Listener_LogsInitialNullStatus_WhenUsingMonitor()
             await RunInitialStatusTestAsync(null, "Function 'Program.TestTimerJob' initial status: Last='', Next='', LastUpdated=''");
         }
 
+        public static IEnumerable<object[]> TimerSchedulesAfterDST => new object[][]
+        {
+            new object[] { new CronSchedule(CrontabSchedule.Parse("0 0 18 * * 5", new CrontabSchedule.ParseOptions() { IncludingSeconds = true })), TimeSpan.FromHours(167) },
+            new object[] { new ConstantSchedule(TimeSpan.FromDays(7)), TimeSpan.FromDays(7) },
+        };
+
+        public static IEnumerable<object[]> TimerSchedulesWithinDST => new object[][]
+        {
+            new object[] { new CronSchedule(CrontabSchedule.Parse("0 59 * * * *", new CrontabSchedule.ParseOptions() { IncludingSeconds = true })), TimeSpan.FromHours(1) },
+            new object[] { new ConstantSchedule(TimeSpan.FromMinutes(5)), TimeSpan.FromMinutes(5) },
+        };
+
         /// <summary>
         /// Situation where the DST transition happens in the middle of the schedule, with the
         /// next occurrence AFTER the DST transition.
         /// </summary>
-        [Fact]
-        public void GetNextInterval_NextAfterDST_ReturnsExpectedValue()
+        [Theory]
+        [MemberData(nameof(TimerSchedulesAfterDST))]
+        public void GetNextInterval_NextAfterDST_ReturnsExpectedValue(TimerSchedule schedule, TimeSpan expectedInterval)
         {
+            Console.WriteLine("Current TimeZone is: " + TimeZoneInfo.Local.ToString());
+            Console.WriteLine("Supports DST: " + TimeZoneInfo.Local.SupportsDaylightSavingTime);
```

Comment:
I added this to see what Appveyor uses -- it uses Pacific time, surprisingly. But that also explains why these tests pass. I'm going to make them throw an exception if the local timezone does not support DST. It would fail anyway, but at least we can give a good explanation then.

Reply:
This fixes the test, but I think we still have a product issue that needs to be addressed.